plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/attribute-filter/inspector.tsx

```diff
@@ -105,7 +105,7 @@ export const Inspector = ( {
 						) }
 					/>
 				</PanelBody>
-				<PanelBody title={ __( 'Settings', 'woocommerce' ) }>
+				<PanelBody title={ __( 'Display', 'woocommerce' ) }>
```

Comment:
I wonder why you opted for using `Display` everywhere instead of `Settings`. To be honest, `Settings` is more abstract and works better for me in all different scenarios.

For example, attributes have a `Logic` setting

![CleanShot 2025-03-17 at 13 59 47](https://github.com/user-attachments/assets/b3176d49-585a-49c2-a48c-7ce00a7bffbb)

Reply:
The reason is it is already the "Settings" Tab. If we used "Settings" again here. It would be "Settings -> Settings". Sounds awkward.

Reply:
Aha, true! That's fair!